app/src/main/java/edu/uc/hornunjb/weatherwear/MainActivity.kt

```diff
@@ -36,7 +36,7 @@ class MainActivity : AppCompatActivity() {
         val repository = Repository()
         val viewModelFactory = MainViewModelFactory(repository)
         viewModel = ViewModelProvider(this, viewModelFactory).get(MainViewModel::class.java)
-        viewModel.getPost()
+        viewModel.getWeather()
```

Comment:
I renamed this method as it is one of the more critical functions of your program, and I felt as though getPost was a bit ambiguous. 

According to https://kotlinlang.org/docs/coding-conventions.html#choose-good-names, the name of a method is usually a verb or a verb phrase saying what the method does. While this method is technically getting a post result from an api, naming it getWeather removes the technical jargon from the name and tell you what the method actually DOES. My mindset is: the more clear and descriptive the method/variable name, the less of a need for a bunch of comments.